src/pages/Register/Register.scss

```
@@ -0,0 +1,62 @@
+@mixin flex($justify: center, $align: center) {
+  display: flex;
+  justify-content: $justify;
+  align-items: $align;
+}
+
+.register-header {
+  min-width: 500px;
+  .register-nav {
+    @include flex;
+    border-bottom: 1px solid #f0f0f0;
+    h1 {
+      padding: 15px 200px;
+      font-size: 20px;
+      font-weight: 500;
+    }
+    button {
+      border: 0;
+      background-color: white;
+      font-size: 20px;
+    }
+  }
+}
+
+.register-body {
+  @include flex;
+  min-width: 500px;
+  .content {
+    margin-top: 40px;
+
+    .inputs {
+      @include flex($align: flex-start);
+      flex-direction: column;
+      input {
+        margin-bottom: 20px;
+        width: 400px;
```

선택자 간에 한 칸씩 띄워서 가독성을 높여주세요!
```suggestion
.register-body {
  @include flex;
  min-width: 500px;
  
  .content {
    margin-top: 40px;

    .inputs {
      @include flex($align: flex-start);
      flex-direction: column;
      
      input {
        margin-bottom: 20px;
        width: 400px;
```

---

참고해서 다른 부분도 수정해주세요!